app/build.gradle

```diff
@@ -27,11 +28,11 @@ android {
     signingConfigs {
         playStore
     }
-    compileSdkVersion 24
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 26
```

Comment:
Right, I've previously looked at this when updating SDK  to 25 for 
 comment bottom sheet. See https://github.com/slapperwan/gh4a/pull/536#issuecomment-290664355 to see my findings. In short, the biggest potential issue is where we pass a large amount of data through Intents as they have a hard limit now. But I don't have ideas how to fix that one....

I'll take another look later and also check for changes to 26 which wasn't released when I tested previously.